packages/vscode-ui5-language-assistant/test/suite/validations/flex-disabled/flex-disabled-spec.ts

```diff
@@ -0,0 +1,60 @@
+import * as vscode from "vscode";
+import { resolve, dirname } from "path";
+import {
+  setFileTextContents,
+  expectProblemView,
+  sleep,
+} from "../../test-utils";
+
+const EXTENSION_START_TIMEOUT = 5000;
+const pkgJsonPath = resolve(
+  __dirname,
+  "..",
+  "..",
+  "..",
+  "..",
+  "..",
+  "package.json"
+);
+
+const rootPkgFolder = dirname(pkgJsonPath);
```

Comment:
again overly complicated path calculation.
- no need for package.json
- no need to traverse so much up and then down

Reply:
maybe extract the common logic of getting the fixture's root to utils so it does not repeat in every test?